### ISSUE TITLE
Pinger not working in hologram but works in papi parse?

### ISSUE THREAD

**Issue by Sean1472** [Niall7459/HolographicExtension#38]
Hey, I've got an issue with using pinger by placeholderAPI in holograms;

As seen here, pinging hypixel in chat works fine;
![yye68ibz](https://user-images.githubusercontent.com/50555026/91490776-e575ac00-e8aa-11ea-818e-b8fa146fed45.png)
but when used in a hologram it seems to just not function at all;
![ejxyuoho](https://user-images.githubusercontent.com/50555026/91490808-f45c5e80-e8aa-11ea-896b-0505140fb6d2.png)

Any way to fix this possibly?

I'm running tuinity 1.15.2

**Comment by Niall7459** [comment]
try `{slow}%pinger_online_mc.hypixel.net:25565%` in the hologram
This will update every 5 seconds

**Comment by Sean1472** [comment]
> try `{slow}%pinger_online_mc.hypixel.net:25565%` in the hologram
> This will update every 5 seconds

I tried this. Didn't seem to change anything. tested {slow} and {fast}. Papi parse still worked fine though.
![baij6iop](https://user-images.githubusercontent.com/50555026/91587499-c3ce0080-e94e-11ea-893b-a086c6fa2131.png)

**Comment by Niall7459** [comment]
Do any other placeholderswork? makesure you have got Protocollib installed as well

**Comment by Sean1472** [comment]
> Do any other placeholderswork? makesure you have got Protocollib installed as well

Yes all other placeholders seem to work fine apart from pining in a placeholder. ProtocolLib is installed.
